packages/react-native-editor/package.json

```diff
@@ -108,14 +108,15 @@
 		"device-tests-canary": "cross-env NODE_ENV=test jest --forceExit --detectOpenHandles --no-cache --maxWorkers=2 --testPathPattern=@canary --config ./jest_ui.config.js",
 		"device-tests:local": "cross-env NODE_ENV=test jest --runInBand --detectOpenHandles --forceExit --config ./jest_ui.config.js",
 		"device-tests:debug": "cross-env NODE_ENV=test node $NODE_DEBUG_OPTION --inspect-brk node_modules/jest/bin/jest --runInBand --detectOpenHandles --verbose --config ./jest_ui.config.js",
+		"appium:setup": "output=`appium driver list --installed --json` && (echo $output | grep -q 'uiautomator2' && echo 'uiautomator2 is installed, skipping installation.' || (echo 'uiautomator2 not found, installing...' && appium driver install uiautomator2)) && (echo $output | grep -q 'xcuitest' && echo 'xcuitest is installed, skipping installation.' || (echo 'xcuitest not found, installing...' && appium driver install xcuitest))",
```

Comment:
This is nice. I noticed this output when I first ran the commands. 

I wonder if we should look to follow the lead of the `wp-scripts` package an place these larger scripts somewhere. We don't need to necessarily do that now — just "thinking out loud."

Reply:
That's a great suggestion! We have some scripts within the `bin` [folder](https://github.com/WordPress/gutenberg/tree/f0b4192b33fc69fe42929ccceda161979435ebc5/packages/react-native-editor/bin) we could move it there and just call the file from here. What do you think?

Reply:
Yeah, placing them in the `bin` folder would be a big improvement over capturing this amount of scripting in `package.json`. It will also make reuse far easier, likely leading to fewer bugs.

Reply:
I've addressed this in https://github.com/WordPress/gutenberg/pull/55257